HelloMaps/src/main/java/com/gluonhq/HelloMaps.java

```diff
@@ -0,0 +1,110 @@
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL GLUON BE LIABLE FOR ANY
+ * DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.gluonhq;
+
+import com.gluonhq.attach.position.Position;
+import com.gluonhq.attach.position.PositionService;
+import com.gluonhq.attach.util.Platform;
+import com.gluonhq.attach.util.Services;
+import com.gluonhq.maps.MapLayer;
+import com.gluonhq.maps.MapPoint;
+import com.gluonhq.maps.MapView;
+import javafx.application.Application;
+import javafx.beans.property.ReadOnlyObjectProperty;
+import javafx.geometry.Pos;
+import javafx.geometry.Rectangle2D;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.Circle;
+import javafx.stage.Screen;
+import javafx.stage.Stage;
+
+public class HelloMaps extends Application {
+
+    private MapPoint mapPoint;
+
+    @Override
+    public void start(Stage stage) {
+        MapView view = new MapView();
+        view.addLayer(positionLayer());
+        view.setZoom(3);
+        Scene scene;
+        if (Platform.isDesktop()) {
+            scene = new Scene(view, 600, 700);
+            stage.setTitle("Gluon Maps Demo");
+        } else {
+            BorderPane bp = new BorderPane();
+            bp.setCenter(view);
+            final Label label = new Label("Gluon Maps Demo");
+            label.setAlignment(Pos.CENTER);
+            label.setMaxWidth(Double.MAX_VALUE);
+            label.setStyle("-fx-background-color: dimgrey; -fx-text-fill: white;");
+            bp.setTop(label);
+            Rectangle2D bounds = Screen.getPrimary().getVisualBounds();
+            scene = new Scene(bp, bounds.getWidth(), bounds.getHeight());
+        }
+        stage.setScene(scene);
+        stage.show();
+
+        view.flyTo(1., mapPoint, 2.);
+    }
+
+    private MapLayer positionLayer() {
+        return Services.get(PositionService.class)
+                .map(positionService -> {
+                    positionService.start();
+
+                    ReadOnlyObjectProperty<Position> positionProperty = positionService.positionProperty();
+                    Position position = positionProperty.get();
+                    if (position == null) {
+                        position = new Position(50., 4.);
```

Comment:
might be better to use constants here? e.g. double DEFAULT_LAT = 50.; (also in the block below this).
That makes it easier for developers to change it to their preferred location (instead of changing it twice inside the code).